packages/integrations/vercel/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@astrojs/vercel",
   "description": "Deploy your site to Vercel",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
Don't change the version number directly, create a changeset.

1. revert this change
2. open the terminal and go to the astro repo
3. run `pnpm changeset`
4. select `@astrojs/vercel` and select a `minor` change
5. write a changeset (you can change it later)
6. commit the file generated